demos/pkcs11/common/include/core_pkcs11_config.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * FreeRTOS V1.4.8
```

Comment:
We have our config files versioned with the CSDK

Reply:
Thanks